### ISSUE TITLE
Some field labels are not unique

### ISSUE THREAD

**Issue by jose-cds** [cds-snc/platform-forms-client#3093]
**HC Assessment Label: Serious**

Pages:  Advancing Accessibility Standards Research Program Expression of Interest
URL: https://forms-staging.cdssandbox.xyz/en/id/134
User agent:  Chrome (v119.0.6045.200)
Language: English and French

Issue Description: 
Under the "Part 2: Project Information" section, the label text for a couple of the form fields are not unique. This could be problematic and confusing for non-sighted users as they are not able to make the visual connection with the instructions above the form fields. For details, see the attached screenshot.

Steps to reproduce:

1. Open the above URL in the Chrome browser
2. Visual check of the content

Expected result: The label text for the fields should be unique, informative, and concise to meet the accessibility standards.

**Comment by srtalbot** [comment]
This is text set by the client and out of our control.
